srcpkgs/mesa-asahi/template

```diff
@@ -1,168 +1,64 @@
 # Template file for 'mesa-asahi'
 pkgname=mesa-asahi
-version=25.1.0+20250425
+version=25.1.1
 revision=1
-_llvmver=19
-_mesaver=${version%+*}
-archs="aarch64*"
-build_style=meson
-build_helper="qemu"
-#Disable LTO flag should be present, see https://gitlab.freedesktop.org/mesa/mesa/-/issues/6911
-configure_args="-Dglvnd=enabled -Dgbm=enabled -Degl=enabled
- -Dgles1=enabled -Dgles2=enabled -Dglx=dri
- -Dlmsensors=enabled -Dplatforms=x11$(vopt_if wayland ,wayland)
- -Dllvm=enabled -Db_lto=false -Dcpp_std=gnu++17
- -Dgallium-vdpau=enabled -Dgallium-va=enabled
- -Dvulkan-drivers=swrast,virtio,asahi -Dvulkan-layers=device-select,overlay
- -Dgallium-drivers=asahi,virgl,zink,llvmpipe
- -Dgallium-rusticl=true -Drust_std=2021"
-hostmakedepends="gettext flex pkg-config python3-Mako glslang llvm${_llvmver}
- $(vopt_if wayland 'wayland-protocols wayland-devel') python3-pycparser python3-yaml
- clang${_llvmver} rust rust-bindgen"
-makedepends="elfutils-devel expat-devel libXdamage-devel
- libXxf86vm-devel libdrm-devel libffi-devel libva-devel
- libvdpau-devel libxshmfence-devel ncurses-devel zlib-devel
- $(vopt_if wayland 'wayland-devel wayland-protocols') llvm${_llvmver}-devel libsensors-devel
- libXrandr-devel libglvnd-devel libzstd-devel libxml2-devel lua53-devel
- libarchive-devel vulkan-loader clang${_llvmver} libclc${_llvmver} rust
- SPIRV-LLVM-Translator${_llvmver}-devel SPIRV-Tools-devel"
-depends="libglvnd"
-short_desc="Open source implementation of OpenGL and Vulkan - Asahi"
+depends="mesa>=${version}_1"
+short_desc="Mesa - Asahi (transitional dummy package)"
 maintainer="dkwo <npiazza@disroot.org>"
-license="MIT, LGPL-2.1-or-later"
+license="Public Domain"
 homepage="https://www.mesa3d.org/"
 changelog="https://docs.mesa3d.org/relnotes.html"
-distfiles="https://gitlab.freedesktop.org/asahi/mesa/-/archive/asahi-${version#*+}/mesa-asahi-${version#*+}.tar.gz"
-checksum=1008a7a245276203a39021d914b4ecc386f33ff50cd872817955c14550a928da
-replaces="mesa>=0 mesa-libgallium>=0 libglapi>=0"
-provides="mesa-${_mesaver}_${revision} mesa-libgallium-${_mesaver}_${revision} libglapi-${_mesaver}_${revision}"
-shlib_provides="libglapi.so libgallium-${_mesaver}-devel.so"
+_metapackage=yes
```

Comment:
xlint is wrong atm, `metapackage=yes` is correct

https://github.com/leahneukirchen/xtools/pull/340